examples/basic/areas/vsteparea.py

```diff
@@ -0,0 +1,11 @@
+from bokeh.plotting import figure, show
+
+p = figure(width=400, height=400)
+
+glyphs = p.vsteparea(
+        x=[1, 2, 3, 4, 5],
+        y1=[12, 16, 14, 13, 15],
+        y2=[1, 4, 2, 1, 3],
+        step_mode='after')
```

Comment:
```suggestion
glyphs = p.vsteparea(
    x=[1, 2, 3, 4, 5],
    y1=[12, 16, 14, 13, 15],
    y2=[1, 4, 2, 1, 3],
    step_mode='after',
)
```

Reply:
Right! Not sure what my vim was thinking here...